### ISSUE TITLE
[Traits] "Missing method" warning on service declaration

### ISSUE THREAD

**Issue by xDaizu** [Haehnchen/idea-php-symfony2-plugin#759]
In my code, this snippet

``` yml
services:
    license.people_manager:
            class: License\PersonBundle\Service\PeopleManager
            calls:
                  - [ setLogger, [@logger] ]
                  - [ setEntityManager, ["@doctrine.orm.entity_manager"] ]
                  - [ setLdapApi, ["@educa.ldap_service"] ]
                  - [ setMoodleApiService, ["@license.moodle_service"] ]
                  - [ setLicenseDomain, ["%license_domain%"] ]
```

puts a warning on each of the parameters (e.g. @logger, "@doctrine.orm.entity_manager","@educa.ldap_service", etc...), marking them as "Missing method". The methods (setLogger, setEntityManager, etc...) DO exist on the class PeopleManager. 

Maybe it's checking the wrong class?

PS: Of course, "@doctrine.orm.entity_manager" shows a double warning, the "Missing method" one, and the "Missing service" one reported on https://github.com/Haehnchen/idea-php-symfony2-plugin/issues/757

**Comment by palra** [comment]
Same here. The method is implemented with a trait, but it's still keeps telling me that the method is missing.

**Comment by Haehnchen** [comment]
duplicate #755

**Comment by cawolf** [comment]
I don't think it's related to traits, it doesn't work even if the methods are implemented directly in the class (PhpStorm 2016.3). So I would question the duplication marking to #755 .

**Comment by Haehnchen** [comment]
indeed not related to traits fixed by #755
